tools/tests/dropdown/dropdown-state.test.js

```diff
@@ -0,0 +1,104 @@
+
+const template = `<gameface-dropdown class="gameface-dropdown-component">
+<dropdown-option slot="option">Cat</dropdown-option>
+<dropdown-option slot="option">Dog</dropdown-option>
+<dropdown-option slot="option">Giraffe</dropdown-option>
+<dropdown-option slot="option">Lion</dropdown-option>
+<dropdown-option slot="option" disabled>Pig</dropdown-option>
+<dropdown-option slot="option">Eagle</dropdown-option>
+<dropdown-option slot="option">Parrot</dropdown-option>
+<dropdown-option slot="option">Last Parrot</dropdown-option>
+</gameface-dropdown>`;
+
+const dropdownSelector = '.dropdown-state-test-wrapper gameface-dropdown';
+
+/**
+ * Method initializing dropdown test page
+ * @param {string} templateString
+ * @returns {Promise<void>}
+ */
+function setupDropdownTestPage(templateString) {
+    const el = document.createElement('div');
+    el.className = 'dropdown-state-test-wrapper';
+    el.innerHTML = templateString;
+
+    cleanTestPage('.dropdown-state-test-wrapper');
+
+    document.body.appendChild(el);
+
+    return new Promise((resolve) => {
+        waitForStyles(resolve);
+    });
+}
+
+// eslint-disable-next-line max-lines-per-function
+describe('Dropdown State', () => {
+    afterAll(() => {
+        cleanTestPage('.dropdown-state-test-wrapper');
+    });

```

Comment:
Why not use `afterEach` instead and remove the `cleanTestPage` in the beforeEach?

Reply:
Done!